generated/gulpfile.js

```diff
@@ -78,6 +82,8 @@ gulp.task('testServerJSWithCoverage', function (done) {
 });
 
 gulp.task('testBrowserJS', function (done) {
+    //testing environment variable 
+    process.env.NODE_ENV = 'testing';
```

Comment:
@omriBernstein @joedotjs I am not sure about frontend tests honestly b/c I haven't really written any, but I thought I might as well throw it in everywhere we are testing